meson_options.txt

```diff
@@ -17,3 +17,4 @@ option('gir-dir',
        value: '',
        description: 'Override the directory where gir-to-d searches for GIR files.'
 )
+option('localbase', type : 'string', value : '/usr', description : 'Prefix under which all 3rd party software is installed')
```

Comment:
Why the name `localbase`?

Reply:
I borrowed the terminology from FreeBSD where `localbase` is a variable (and a sysctl) that contains "path to the local software directory". I'm not really sure what "local" means in this context.